TSSSF_CardGen.py

```diff
@@ -318,9 +320,16 @@ def AddCardArt(image, filename, anchor):
         art = random.choice(ArtMissing)
     # Find desired height of image based on width of 600 px
     w, h = art.size
-    h = int((float(ART_WIDTH)/w)*h)
-    # Resize image to fit in frame
-    art = PIL_Helper.ResizeImage(art, (ART_WIDTH,h))
+    if h > w:
```

Comment:
I could be wrong, but this looks like it's testing how rectangular the art is, rather than if it meets the expected aspect ratio.